### PR TITLE
Hot Road

### PR DESCRIPTION
**Pull request by PBWiseman**
## Describe your changes

Adding a hot road hazard that slows down movement and increases in intensity as the game progresses.

## Issue number and link

#37 

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by PBWiseman** [comment]
Floor tile with a hot road tag will slow you down at an increased value throughout the match. It will also stop sliding. This can be tested in the Australia 1 level

File: Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -29,6 +58,11 @@ public class PlayerMovement : MonoBehaviour
             if (IsMoving)
             {
                 //Moving Speed
+                if (touchingRoad)
+                {
+                    Debug.Log("Touching Road Speed: " +  (MOVEMENTSPEED - (int)TouchingRoadSpeed));
```
Comment (ChaseBENNC9) [inline Assets/Scripts/PlayerMovement.cs:63]: This debug statement could be removed

**Review by ChaseBENNC9 — CHANGES_REQUESTED** [review]
# Code Review

## Functional Requirements
_Is the application fit for purpose? Does it meet the user’s requirements and properly address the issue?_  

 -  meets all functional requirements
 -  players are slowed down by the road
 - the amount that the player is slowed down by increases over time
 - sliding onto the road is cancelled out


## Legibility and Style
_Uses agreed coding standards/conventions? Readable, consistent and well-structured with useful comments?_  

 - Excellent use of code comments. all code has been formatted 
_Any repetitive code that could be refactored? (DRY)_  

 - code is DRY


## Correctness of the Code
_Is the code reliable, giving same results every time it runs? Does it function correctly?_  

 - (EnterTextHere)


## Implementation
_Any notable performance/speed issues? Compatibility and Responsiveness on other platforms ?_  

 - Not that i could find


# Final Comments

 - Only thing was a small tidy up to remove the debug log. then it should be ready to merge

***

**Review by erokasyuart — APPROVED** [review]
Looks good!

File: Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (PBWiseman) [reply Assets/Scripts/PlayerMovement.cs]: Removed

**Review by ChaseBENNC9 — APPROVED** [review]

**Pull request merged**